hudi-sync/hudi-sync-common/src/main/java/org/apache/hudi/sync/common/operation/CatalogSync.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.hudi.sync.common.operation;
```

Comment:
@fengjian428 you'd need to configure IDE to auto add license to source files see https://hudi.apache.org/contribute/developer-setup